tests/acceptance/test_organization_events_v2.py

```diff
@@ -375,7 +375,6 @@ def test_event_detail_view_from_transactions_query(self, mock_now):
         child_event["transaction"] = "z-child-transaction"
         child_event["spans"] = child_event["spans"][0:3]
         self.store_event(data=child_event, project_id=self.project.id, assert_no_errors=True)
-        self.wait_for_event_count(self.project.id, 2)
```

Comment:
We can't `wait_for_event_count()` on transactions since this function only checks the events table, and transactions are no longer dual stored there